TouchVisualizer/UIWindow+Swizzle.swift

```diff
@@ -8,8 +8,8 @@ import UIKit
 fileprivate var isSwizzled = false
 
 @available(iOS 8.0, *)
-extension UIWindow {
-	
+extension UIApplication {
```

Comment:
Let's rename the file as well.

Reply:
[UIAppilcation + Swizzle file name change](https://github.com/bandlab/TouchVisualizer/pull/1/commits/da2bbfe5bbb4be8ba01df32dd9d4274092be7c4f)